packages/kernel/src/kernel.ts

```diff
@@ -354,7 +354,7 @@ export class StliteKernel {
         data: payload,
       },
       "reply:language-server:code_completion",
-    ).then((data) => data);
+    ).then((data) => ({ type: "reply:language-server:code_completion", data }));
```

Comment:
Can you revert this change, because the `ReplyMessageLanguageServerCodeCompletion` is a type only for the inter-process messaging so I don't want it to be exposed?

Then the message object for the sharing will be created in its code (`packages/sharing/src/App.tsx`) separately even if the `type` value is the same.

Reply:
Ok, I will look into it

Reply:
Done